alias/config.c

```diff
@@ -0,0 +1,54 @@
+#include "config.h"
```

Comment:
Can you add a Copyright header block here.
We also need a Doxygen @page block, which needs linking up in `alias/lib.h`

Reply:
Sorry! My bad.

Done! :-)